packages/server/build.gradle

```diff
@@ -111,14 +111,6 @@ tasks.named('test') {
     useJUnitPlatform()
 }
 
-hibernate {
-    enhancement {
-        lazyInitialization true
-        dirtyTracking true
-        associationManagement true
```

Comment:
This seems to control objects that contain other objects and how they get written to the database.  I seriously do not understand why devs don't just learn SQL and understand how this all works instead of trusting a monstrosity of automajik code

Reply:
Yeah I This is the source of the extra rows. When enabled it allows you to only set one side of the relationship and it updates any other in memory references for you. When its off, you need to explicitly insert it into the other side if you want it there, or re-fetch it from the db. We've already had this logic in for project assets and now the notifications to get the tests to originally pass, so I'm not too worried it will cause issues. It doesn't affect what was written to the db, just what is already held in memory in Java.